recipes/hoomd-dlext/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set processor = "cpu" %}
+{% set name = "hoomd-dlext" %}
+{% set version = "0.2.1" %}
+{% set build = 0 %}
+{% set sha256 = "547316c95608d7ee1ca11ace53ccf22dddc88206fa0ac0613020f48bf6081269" %}
+{% set processor = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux64]
+{% set processor = "cpu" %}  # [not linux64]
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SSAGESLabs/hoomd-dlext/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  # Upstream package does not support Windows, Python < 3.6, or PyPy
+  skip: true  # [win or py<36 or python_impl == "pypy"]
+  string: {{ processor }}_py{{ py }}h{{ PKG_HASH }}_{{ build }}
+  ignore_run_exports: tbb
```

```suggestion
{% set version = "0.2.1" %}
{% set build = 0 %}

package:
  name: hoomd-dlext
  version: {{ version }}

source:
  url: https://github.com/SSAGESLabs/hoomd-dlext/archive/refs/tags/v{{ version }}.tar.gz
  sha256: 547316c95608d7ee1ca11ace53ccf22dddc88206fa0ac0613020f48bf6081269

build:
  number: {{ build }}
  # Upstream package does not support Windows, Python < 3.6, or PyPy
  skip: true  # [win or py<36 or python_impl == "pypy"]
  string: {{ 'cuda' ~ cuda_compiler_version|string }}_py{{ py }}h{{ PKG_HASH }}_{{ build + 100}}  # [cuda_compiler_version != "None"]
  ignore_run_exports_from:
    - tbb-devel
```
Only use jinja variables for values will change or are used in multiple places